### PR TITLE
Created a BuildLegend sample for widgets.

### PR DESCRIPTION
**Pull request by anmacdonald**
Created a BuildLegend sample to demonstrate Qt Widgets integration.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples_Widgets/DisplayInformation/BuildLegend/BuildLegend.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+public:
+  explicit BuildLegend(QWidget* parent = nullptr);
+  ~BuildLegend();
+
+private:
+  void addLayers();
+
+  Esri::ArcGISRuntime::Map* m_map = nullptr;
+  Esri::ArcGISRuntime::MapGraphicsView* m_mapView = nullptr;
+  Ui::BuildLegend* m_ui;
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples_Widgets/DisplayInformation/BuildLegend/BuildLegend.h]: is it best practice to initialize this as nullptr?

**Comment by ldanzinger** [comment]
@anmacdonald I think there might be a small issue with the text color with the alternating background colors

screen shot 2018-10-31 at 1 39 52 pm

**Comment by anmacdonald** [comment]
@ldanzinger is that you running dark theme on a Mac? I hadn't even considered that possibility. Will see what I can do.

**Comment by anmacdonald** [comment]
https://bugreports.qt.io/browse/QTBUG-68891

This may be a general widgets issue until 5.12. If you think it is worthwhile I can work around it by including a qss file.

**Comment by ldanzinger** [comment]
@anmacdonald yep I am using dark mode. I think this is good to merge

**Pull request merged**